pkgs/by-name/se/seadrive-gui/fix_webkit_webengine.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c952008..f4afb47 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -181,6 +181,7 @@ ENDIF()
+ 
+ SET(USE_QT_LIBRARIES
+     Core Gui Widgets LinguistTools Network Test
++    ${WEBKIT_NAME} ${WEBKIT_WIDGETS_NAME}
```

Comment:
Any chance this could be PR'd upstream and then `fetchpatch`'d in nixpkgs? (rather than vendoring non-nixpkgs-specific patch files in nixpkgs)

Reply:
A PR does not have to be merged to be `fetchpatch`'d from nixpkgs (this is common and is a way I deal with unresponsive upstreams)

If y'all aren't comfy opening a PR, I can open one myself